doc/releases/changelog-dev.md

```diff
@@ -26,6 +26,10 @@
 
 <h3>Documentation 📝</h3>
 
+* Add functions for qubit-simulation to the `qml.devices` sub-page of the "Internal" section.
+  Note that these functions are unstable while device upgrades are underway.
```

Comment:
This note might be unnecessary by the time 0.33 rolls around.

Reply:
We probably could start to lock them down. They are handy enough that we should think about it, especially for `apply_operation`.

Reply:
I agree that it probably won't be necessary by then. If we stabilize them, I'll remove this line.